tests/integration/ambient/testdata/wasm-filter-service.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: extensions.istio.io/v1alpha1
+kind: WasmPlugin
+metadata:
+  name: {{ .WasmPluginName }}
+  {{- if .WasmPluginVersion }}
+  annotations:
+    wasmPluginVersion: {{ .WasmPluginVersion }}
+  {{- end }}
+spec:
+  phase: STATS
+  targetRef:
+    kind: Service
```

Comment:
except for the targetRefs part, all the wasm filters are same. May be you just need a single template file and it can be rendered based on the testcase specific arguments.

Reply:
done